conversion.c

```diff
@@ -65,16 +57,16 @@ char* Binary (int input) {
 	int digit;
 	long int power;
 	bool largestNum = 0;                // Flag sets if the number is the largest within the given byte length ex. 256,-255
-	unsigned num = (unsigned)input;
-	digit = Digits (num, 2);
-	if (digit < 0) return NULL;
+	unsigned num = (unsigned)input,numCpy = num;
+	for (digit = 0; numCpy > 0; digit++) numCpy /= 2;
+	if (digit > 32) return NULL;
 	power = pow (digit, 2);
 	if (num == (power - 1)) {
```

Comment:
The variable power is only used in this place. So, the inbuilt function pow() can be directly used in this place.
if (num == (pow(digit,2) - 1))